README.md

```diff
@@ -77,7 +77,18 @@ https://www.googleapis.com/auth/admin.directory.user,
 ```
 
 Now that you have a credential that is allowed to the Admin SDK, you can use the
-GSuite provider.
+G Suite provider.
+
+## Configuration
```

Comment:
I think this still fits in the `Authentication` block as `Customer ID`

Reply:
Removed because instead of the actual customer ID it is possible to use `my_customer`, which would be the G Suite customer ID associated with the impersonating account. This is also specified in https://developers.google.com/admin-sdk/directory/v1/guides/manage-customers#update_an_existing_customer.